### ISSUE TITLE
Avatar - Update guidance

### ISSUE THREAD

**Issue by KennyAtHPE** [grommet/hpe-design-system#2988]
Guidance here
Related to #2982 

### Deliverables
- [x] Replace Card example with a different medium-sized Avatar use case
- [x] Include 'When not to use' section
      - When to use icons or a box instead
- [x] Include guidance on when to include a white background to an Avatar
      - Create a dark mode example
- [x] Provide additional guidance on why an Avatar should be used
      - Predefined-sized box to provide a consistent structure

**Comment by taysea** [comment]
@KennyAtHPE I took a stab at these changes here: https://www.figma.com/file/GMvJnEVPSz22ZlfOKzItY9/Untitled?node-id=0%3A1

I don't have edit access to the avatar file so feel free to copy this over to that WIP file if you'd like and make edits as you see fit.

**Comment by KennyAtHPE** [comment]
Left some comments. Also created a new version based on some of my thoughts and left comments as well. Check everything out [here ](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=1686%3A2532)

Sent out an invite so lets connect on this later today @taysea @britt6612

**Comment by KennyAtHPE** [comment]
Good to close out once #2933 is finished

**Comment by KennyAtHPE** [comment]
Guidance has been finalized, followup ticket is #2933
